ci/aarch64.conda_requirements.txt

```diff
@@ -1,5 +1,5 @@
-lockfile >= 0.10.2
-cachecontrol >= 0.11.5
+filelock >= 3.8.0
+cachecontrol >= 0.13.0
```

Comment:
@ebolyen, do you anticipate these new minimal requirements presenting issues with QIIME 2 builds?